### ISSUE TITLE
Freezer flag not honored when creating product

### ISSUE THREAD

**Issue by luismanson** [grocy/grocy#651]
Hello, i'm running latest version, and noticed that adding defined products located in a freezer are added with its default expiry time, and the 'days after freezing' days are not added. This might be a bug, moving this product out and in from the freezer gets the right amount of days added up.

Also, I think it would be best to allow áéíóúñ to be the same as aeioun in the search engine.   

Thanks for the app, its really useful! :)

**Comment by berrnd** [comment]
This was defined to work like this, excerpt from the changelog of v2.6.0:

> ### New feature: Freeze/Thaw products
>- New product options "Default best before days after freezing/thawing" to set how the best before date should be changed on freezing/thawing
>- New location option "Is freezer" to indicate if the location is a freezer
>- => When moving a product from/to a freezer location, the best before date is changed accordingly
> - There is also a new sub feature flag `FEATURE_FLAG_STOCK_PRODUCT_FREEZING` to disable this if you don't need it (defaults to `true`)

Your idea would be general more like "Default best before days" per location (and of course somehow per product). Feel free to request this in a separate issue when needed.

> Also, I think it would be best to allow áéíóúñ to be the same as aeioun in the search engine.

Search is entirely done by DataTables (https://datatables.net/reference/api/search()), if you see there options on how to do/configure this, let me know or create a pull request.

**Comment by luismanson** [comment]
> Search is entirely done by DataTables (https://datatables.net/reference/api/search()), if you see there options on how to do/configure this, let me know or create a pull request.

I've found [a plugin](https://datatables.net/plug-ins/filtering/type-based/accent-neutralise#Version-control).
Are you aware of [Tabulator](http://tabulator.info/examples/4.5), Check it out if you have time. I think it might be a better fit. 

> Your idea would be general more like "Default best before days" per location (and of course somehow per product). Feel free to request this in a separate issue when needed.

Sorry, I don't fully understand your answer, you don't see this as a bug, you will be fixing this with a new feature or I don't get the usage flow of the freezer feature. 
If i buy raw chicken, I create and store the product directly in the freezer location, and since it has the freezer flag set, I think the products have to be added with the additional amount of days added. 
Yet your idea to make this some kind of feature sounds very useful


Thanks.

**Comment by berrnd** [comment]
> I've found [a plugin](https://datatables.net/plug-ins/filtering/type-based/accent-neutralise#Version-control).

Thanks, I created #654 to track this.

> Are you aware of [Tabulator](http://tabulator.info/examples/4.5), Check it out if you have time. I think it might be a better fit. 

Also thanks for that hint, will maybe take care of it immediately after the 100+ other requests are done... ;D

For the rest of the main topic here:
On purchase, the best before date is prefilled by the products "Default best before days". The hints next to the product options "Default best before days after freezing" and "Default best before days after thawing" clearly state that this only applies on transfering products to/from a freezer location. So yes, I don't think this is a bug, it works like described, and this was based on the discussions about adding that feature.

I prefer to add things in a general way, so I created #655 to track that.
